### PR TITLE
add multiple account support for env config

### PR DESCRIPTION
**Pull request by futuretech6**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -232,11 +232,22 @@ function determineRunType() {
 }
 
 function getConfigFromEnv() {
+    const usernameArray = (process.env.ZJU_USERNAME || reportFatal('ZJU_USERNAME not set')).split(",")
```
Comment (AsterNighT) [inline src/main.ts]:
```suggestion
    const usernameArray = (process.env.ZJU_USERNAME || reportFatal('ZJU_USERNAME not set')).split(",").map(s => s.trim());
```

File: src/main.ts [hunk 2]
```diff
@@ -232,11 +232,22 @@ function determineRunType() {
...
+    const passwordArray = (process.env.ZJU_PASSWORD || reportFatal('ZJU_PASSWORD not set')).split(",")
```
Comment (AsterNighT) [inline src/main.ts]:
```suggestion
    const passwordArray = (process.env.ZJU_PASSWORD || reportFatal('ZJU_PASSWORD not set')).split(",").map(s => s.trim());
```

File: src/main.ts [hunk 3]
```diff
@@ -232,11 +232,22 @@ function determineRunType() {
...
+
+    if (usernameArray.length != passwordArray.length) {
+        console.error("username and password length mismatch, please use comma ',' without whitespace")
```
Comment (AsterNighT) [inline src/main.ts]:
```suggestion
        console.error("username and password length mismatch");
```

File: src/main.ts [hunk 4]
```diff
@@ -232,11 +232,22 @@ function determineRunType() {
...
+    }
+    let accountArray = new Array();
```
Comment (AsterNighT) [inline src/main.ts]:
```suggestion
    const accountArray = usernameArray.map((username,i) => ({username, password: passwordArray[i]});
```

**Review by AsterNighT — CHANGES_REQUESTED** [review]
Just a few tweaks

File: src/main.ts [hunk 5]
```diff
@@ -232,11 +232,22 @@ function determineRunType() {
 }
 
 function getConfigFromEnv() {
+    const usernameArray = (process.env.ZJU_USERNAME || reportFatal('ZJU_USERNAME not set')).split(",")
+    const passwordArray = (process.env.ZJU_PASSWORD || reportFatal('ZJU_PASSWORD not set')).split(",")
+
+    if (usernameArray.length != passwordArray.length) {
```
Comment (AsterNighT) [inline src/main.ts]:
```suggestion
    if (usernameArray.length !== passwordArray.length) {
```

**Comment by futuretech6** [comment]
> Just a few tweaks

not familiar with js 😅

**Review by AsterNighT — APPROVED** [review]
LGTM

**Pull request merged**